core/docker/importer.sh

```diff
@@ -1,2 +1,23 @@
+#!/bin/sh
 
-curl -X POST --insecure -u "$OPENHIM_API_USERNAME:$OPENHIM_API_PASSWORD" -H "Content-Type: application/json" -d @openhim-import.json "$OPENHIM_API_SERVER/metadata" -v
\ No newline at end of file
+OPENHIM_RESPONSE="";
+TIMEOUT=0
+
+while [ "$OPENHIM_RESPONSE" != "200" ]
+do
+  TIMEOUT=$((TIMEOUT+1))
+  # Fail script if incomplete at 30 seconds
+  if [ "$TIMEOUT" == "15" ]
```

Comment:
The comment and the actual condition doesnt seem to match up. The comment states 30 seconds but the condition for the timeout is set at 15 seconds.

Assuming this has to do with the `sleep 2` which would equate to 30 seconds (15 iterations * 2 seconds sleeping). This however doesnt take into account the time it takes for the actual curl request to be completed.

Maybe this should be actual time based and not based on iterations

Reply:
I'm wondering if we should use a generic utility for this so we don't have to worry too much about this logic. See my comment below.